### PR TITLE
[improvement](pipeline) ADD LOG in analyze task

### PR DESCRIPTION
**Pull request by Kikyou1997**
1. ADD LOG in analyze task

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Kikyou1997** [comment]
run buildall

**Comment by Kikyou1997** [comment]
run buildall

File: regression-test/suites/brown_p2/load.groovy [hunk 1]
```diff
@@ -72,6 +72,6 @@ suite("load") {
                 assertTrue(json.NumberLoadedRows > 0 && json.LoadBytes > 0)
             }
         }
-        sql """ ANALYZE TABLE $tableName WITH SYNC """
```
Comment (englefly) [inline regression-test/suites/brown_p2/load.groovy:75]: will this change make case unstable?

File: fe/fe-core/src/main/java/org/apache/doris/statistics/OlapAnalysisTask.java [hunk 2]
```diff
@@ -122,6 +123,7 @@ public void execSQL(String sql) throws Exception {
                 throw new RuntimeException(String.format("Failed to analyze %s.%s.%s, error: %s sql: %s",
                         info.catalogName, info.dbName, info.colName, sql, queryState.getErrorMessage()));
             }
+            LOG.info("Analyze SQL: " + sql + " cost time: " + (System.currentTimeMillis() - startTime) + "ms");
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/OlapAnalysisTask.java:126]: log debug

**Comment by Kikyou1997** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Comment by Kikyou1997** [comment]
run buildall

**Review by sohardforaname — APPROVED** [review]

**Pull request merged**